aggregate_root/lib/aggregate_root.rb

```diff
@@ -7,7 +7,7 @@ module AggregateRoot
   def apply(*events)
     events.each do |event|
       apply_strategy.(self, event)
-      unpublished_events << event
+      unpublished << event
```

Comment:
totally minor, but the previous name was more explicit. might be useful for new contributors

Reply:
We now have public api `unpublished_events` which is read_only and private API `unpublished` which does `||=` when needed. I don't know what previous name you have in mind. We can't use `unpublished_events` anymore because it is read only

Reply:
Can we use `@unpublished_events` for internal use, and `unpublished_events` for the public accessor? I think that would be idiomatic Ruby to solve this kind of a problem.

Reply:
We could be we would need to repeat the `||=` logic. I don't see anything un-idiomatic with current way.